datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -1815,6 +1815,8 @@ fn round_trip_datatype() {
     }
 }
 
+// TODO file a ticket about handling deprecated dict_id attributes
```

Comment:
- filed https://github.com/apache/datafusion/issues/14173

I also updated this comment in
- https://github.com/apache/datafusion/pull/14174/files